.travis-script.sh

```diff
@@ -1,48 +1,56 @@
-sudo apt-get --yes update
-sudo apt-get --yes install python-pip
-sudo pip install enum34
-sudo pip install nose
+# All apt-get installs
+sudo apt-get --yes update # Update
```

Comment:
Are the comments of the next 4 lines a bit redundant, perhaps?

Reply:
Yup. They seem some what redundant. Shall I remove them then ?

Reply:
Yes please